packages/template-ui/src/components/ChannelFooter.vue

```diff
@@ -11,6 +11,12 @@
           </div>
           <img :src="logo" aria-hidden="true">
         </div>
+        <div v-else-if="isEndlessCurated" class="endless-seal text-right">
```

Comment:
If we put that text in options.json, we'll need to think about localization in there, which wouldn't be ideal, but we could at least have this be one consistent option instead of two different ones :)

It sounds like these labels are due to be fiddled with in the near future, but let's ask tomorrow to see if there's a better way to implement them…

Reply:
Agreed! I wouldn't advice on adding text to the options.json. Maybe a component override is better in this case. But first let's ask Joana.

Reply:
Okay, based on today's discussion I'll get rid of `isEndlessCurated` and use the footer card we already have, on the assumption there will be new copy in the future that makes sense in this case :)